.gitignore

```
@@ -0,0 +1,2 @@
+build
```

I believe you'll want to constrain this a little more -- I believe this will ignore any directories that include a `build` path component. Based on the lit config below, I'm guessing this should be:

```suggestion
/build
```

---

Valid point; just changed! Thanks.